webpack/lib/node/NodeOutputFileSystem removed in Wepback 5

* Operating System: Mac 10.14.5
* Node Version: 8.9 _(but it's not relevant to this issue)_
* NPM Version: Yarn 1.15.2
* webpack Version: 5.0.0-alpha.17
* webpack-dev-middleware Version: 3.7.0

### Expected Behavior

Don't crash when starting a Webpack dev server.

### Actual Behavior

```
Error: Cannot find module 'webpack/lib/node/NodeOutputFileSystem'
Require stack:
- ../node_modules/webpack-dev-middleware/lib/fs.js
- ../node_modules/webpack-dev-middleware/index.js
- ../node_modules/webpack-dev-server/lib/Server.js
```

### Code

```js
const NodeOutputFileSystem = require('webpack/lib/node/NodeOutputFileSystem');

const { mkdirp } = new NodeOutputFileSystem();
```

`mkdirp` in `NodeOutputFileSystem` is directly `require("mkdirp")` per #405. I think this issue would be fixed if `fs.js` directly had `const mkdirp = require("mkdirp");` instead.

### How Do We Reproduce?

Try out the Webpack 5 alpha: https://github.com/webpack/webpack/issues/8537

Issue reported on that thread: https://github.com/webpack/webpack/issues/8537#issuecomment-504672759

--- a/lib/fs.js
+++ b/lib/fs.js
@@ -4,12 +4,10 @@ const fs = require('fs');
 const path = require('path');
 
 const MemoryFileSystem = require('memory-fs');
+const mkdirp = require('mkdirp');
 const { colors } = require('webpack-log');
-const NodeOutputFileSystem = require('webpack/lib/node/NodeOutputFileSystem');
 
 const DevMiddlewareError = require('./DevMiddlewareError');
-
-const { mkdirp } = new NodeOutputFileSystem();
 
 module.exports = {
   toDisk(context) {

--- a/package.json
+++ b/package.json
@@ -33,6 +33,7 @@
   "dependencies": {
     "memory-fs": "^0.4.1",
     "mime": "^2.4.2",
+    "mkdirp": "^0.5.1",
     "range-parser": "^1.2.1",
     "webpack-log": "^2.0.0"
   },

